src/content/community/online/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Online communities
+description: A listing of the grant programs throughout the Ethereum ecosystem.
+sidebar: true
+lang: en
+---
+
+# Online communities {#online-communities}
+
+Hundreds of thousands of Ethereum enthusiasts gather in these online forums to share news, talk about recent developments, debate technical issues, and imagine the future.
+
+## Forums
+
+<!-- - ![POAP Logo](src/assets/poap-logo.svg) POAP -->
+
+- [r/ethereum](https://www.reddit.com/r/ethereum/) - _all things Ethereum_ <Icon name="youtube" />
+- [r/ethfinance](https://www.reddit.com/r/ethfinance/) - _the financial side of Ethereum, including DeFi_
+- [r/ethdev](https://www.reddit.com/r/ethdev/) - _focused on Ethereum development_
+- [r/ethtrader](https://www.reddit.com/r/ethtrader/) - _trends & market analysis_
+- [r/EtherMining](https://www.reddit.com/r/EtherMining/) - _focused on securing the Ethereum Network (mining)_
+- [r/ethstaker](https://www.reddit.com/r/ethstaker/) - _welcome to all interested in staking on Ethereum_
+- [Fellowship of Ethereum Magicians](https://ethereum-magicians.org) - _community oriented around technical standards in Ethereum_
+- [Ethereum Stackexchange](https://ethereum.stackexchange.com) - _discussion and help for Ethereum developers_
+- [Ethereum Research](https://ethresear.ch) - _the most influential messageboard for cryptoeconomic research_
+
+## Chat rooms
+
+- <Icon name="discord" /> <a href="https://gitter.im/ethereum/home">Ethereum Gitter</a> - <i>chat room for the Ethereum GitHub repo</i>
```

Comment:
I'm not sure we should link out to these Gitter channels anymore. Most of the ones I checked out seem outdated, unmoderated & full of scams.